pkg/mimirtool/commands/analyse_prometheus.go

```diff
@@ -98,15 +99,18 @@ func (cmd *PrometheusAnalyzeCommand) run(k *kingpin.ParseContext) error {
 		jobCount   map[string]int
 	}{}
 	inUseCardinality := 0
-
+	var errorMetrics []string
 	for _, metric := range metricsUsed {
 		ctx, cancel := context.WithTimeout(context.Background(), cmd.readTimeout)
 		defer cancel()
 
 		query := "count by (job) (" + metric + ")"
 		result, _, err := v1api.Query(ctx, query, time.Now())
 		if err != nil {
-			return errors.Wrap(err, "error querying "+query)
+			errStr := fmt.Sprintf("error querying %v: %s", query, err.Error())
```

Comment:
I would clarify the analysis for the given metric will be missing in the output.

```suggestion
			errStr := fmt.Sprintf("skipped %s analysis because failed to run query %v: %s", metric, query, err.Error())
```